project.yaml

```diff
@@ -992,7 +1011,7 @@ Layer:
                 foot,
                 bicycle,
                 tracktype,
-                'null',
+                'null' AS surface, -- Should be a SQL NULL?
```

Comment:
There has to be a column as part of that query as it's part of a JOIN. The question isn't why there's a colum, but if it should be `'null'::text` or `NULL::text`